python/servo/testing_commands.py

```diff
@@ -338,7 +338,7 @@ def test_webidl(self, quiet, tests):
         return run_globals["run_tests"](tests, verbose)
 
     @Command('test-wpt-failure',
-             description='Run the web platform tests',
+             description='Run the small test harness that verifies that failures in tests are reported correctly',
```

Comment:
Let's go for something like, `"Run the test harness that verifies that the test failures are reported correctly"`?